### DISCUSSION TITLE
2023fall%20CS%20PhD%20%E7%BE%8E%E5%9B%BD%E7%94%B3%E8%AF%B7%E6%80%BB%E7%BB%93%20by%20Flash/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#126] [Announcements]
# 2023fall%20CS%20PhD%20%E7%BE%8E%E5%9B%BD%E7%94%B3%E8%AF%B7%E6%80%BB%E7%BB%93%20by%20Flash/

https://csmsapp.org/2023fall%20CS%20PhD%20%E7%BE%8E%E5%9B%BD%E7%94%B3%E8%AF%B7%E6%80%BB%E7%BB%93%20by%20Flash/

**Comment by OvertheBrain** [comment]
flash🤤🤤

**Comment by jhzhang07** [comment]
flash😍😍
